client/coral-framework/graphql/queries/myCommentHistory.graphql

```diff
@@ -0,0 +1,9 @@
+query myCommentHistory {
```

Comment:
What is the advantage of breaking these into their own files? It seems to be that having one file per query containing the fragment and the wrapper would be encapsulate each query/mutation cleanly and minimize the number of imports/files.